clusterloader2/api/types.go

```diff
@@ -106,6 +109,19 @@ type ListUnknownObjectOptions struct {
 	LabelSelector *metav1.LabelSelector `json: labelSelector`
 }
 
+// NamespaceConfig defines namespace parameters.
+type NamespaceConfig struct {
+	// Count is a number of automanaged namespaces.
+	Count *int32 `json: count,omitempty`
```

Comment:
Let's remove the pointer here.
Basically, "0" is the bad value (the test doesn't make sense for count=0). So we don't really need to distinguish it from nil - we can simply default if set to 0.